### PR TITLE
Tests for server.

### PR DESCRIPTION
**Pull request by tomusdrw**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -45,7 +46,7 @@ const tracks = {
 
 app.post('/push-release/:tag/:commit', handleAsync(async function (req, res) {
 	if (keccak256(req.body.secret || '') !== secretHash) {
-		throw new Error('Bad request');
+		throw new Error('Invalid secret');
```
Comment (ngotchac) [inline server.js]: Can this bit be a middleware if it's used several times?

Comment (ngotchac) [inline server.js:49]: Can this bit be a middleware if it's used several times?

**Review by ngotchac — APPROVED** [review]
Looks about right.

File: server.js (hunk 1 above)
Comment (tomusdrw) [reply server.js:49]: Had the same idea, realized in https://github.com/paritytech/push-release/pull/8

**Pull request merged**